### PR TITLE
Add two more address lines

### PR DESCRIPTION
**Pull request by jbalboni**
Closes [491](https://github.com/department-of-veterans-affairs/healthcare-application-team/issues/491)

Adds two more lines to the Address component and (I think) makes the backend changes to accommodate that.

screen shot 2016-11-03 at 4 07 00 pm

### CODE REVIEW

**Comment by jbalboni** [comment]
@akainic @markolson Can you two take a look at this? I'm not super confident in the backend changes, since I don't really know how to test them well.

File: test/data/conformance/australian-vet.xml [hunk 1]
```diff
@@ -28,6 +28,8 @@
             <eeSummary:city>Sydney</eeSummary:city>
             <eeSummary:country>AUS</eeSummary:country>
             <eeSummary:line1>123 NW 8th St</eeSummary:line1>
+            <eeSummary:line2></eeSummary:line2>
```
Comment (markolson) [inline test/data/conformance/australian-vet.xml]: Based on how we treat null/non-existant elements elsewhere, I'm thinking that these shouldn't be in the generated XML if they're blank

Comment (jbalboni) [inline test/data/conformance/australian-vet.xml]: @markolson Should be fixed now.

**Review by markolson — APPROVED** [review]

**Comment by jbalboni** [comment]
@akainic Since @markolson is going to be out, do you want to hold off on merging this? Or should I go ahead when CI is done?

**Comment by markolson** [comment]
@jbalboni merge away, I'd say.

**Pull request merged**